main.py

```diff
@@ -470,21 +475,25 @@ def _send_flow_mods_from_request(self, dpid, command, flows_dict=None):
             f"Send FlowMod from request dpid: {dpid} command: {command}"
             f" flows_dict: {flows_dict}"
         )
-        if dpid:
```

Comment:
I haven't changed the logic, but since I was at it, took the opportunity to flatten these nested logic conditionals.

Reply:
great solution! thanks Vinicius.